src/middlewared/middlewared/plugins/apps/resources.py

```diff
@@ -112,6 +113,15 @@ async def ip_choices(self):
             for ip in await self.middleware.call('interface.ip_in_use', {'static': True, 'any': True})
         }
 
+    @accepts(roles=['CATALOG_READ'])
+    @returns(Int())
+    async def available_space(self):
+        """
+        Returns space available in bytes in the configured apps pool which apps can consume
+        """
+        await self.middleware.call('docker.state.validate')
+        return (await self.middleware.call('filesystem.statfs', IX_APPS_MOUNT_PATH))['free_bytes']
```

Comment:
This is a nit but we should use the `avail_bytes` key since this an `available_space` endpoint.